slurm/pi.py

```diff
@@ -31,23 +33,32 @@ def estimate_pi(iterations, seed, threads=1):
         pool = Pool(processes=threads)
 
         gen = random.Random(seed)
-        seeds = [gen.randint(0, 2**32-1) for _ in range(threads)]
+        seeds = [gen.randint(0, 2**32 - 1) for _ in range(threads)]
         iters = [iterations_per_worker]*threads
         in_circle_points = pool.map(pic_wrapper, zip(iters, seeds))
 
         pool.close()
 
-        # Total number of in-circle points
-        return sum(in_circle_points)*4/sum(iters)
+        # Returns Pi and in-circle points (successes)
+        return sum(in_circle_points)*4/sum(iters), sum(in_circle_points)
     else:
-        return points_in_circle(iterations, seed)*4/iterations
+        return points_in_circle(iterations, seed)*4/iterations, \
+                points_in_circle(iterations, seed)
```

Comment:
Isn't this running the calculation twice?

Reply:
Yes. I missed that.